api/src/main/resources/api/service_openapi.yaml

```diff
@@ -180,6 +180,43 @@ paths:
         200:
           $ref: "#/components/responses/SearchEntityCountsResponse"
 
+  #########################UNDERLAY CONFIG V2 [START]#################################
+  "/v2/underlays":
+    get:
+      summary: List the underlays
+      operationId: listUnderlaysV2
+      tags: [UnderlaysV2]
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UnderlayListV2"
+        500:
+          $ref: '#/components/responses/ServerError'
+
+  "/v2/underlays/{underlayName}":
+    parameters:
+      - $ref: "#/components/parameters/UnderlayNameV2"
```

Comment:
What is the point of having a reference here instead of including it inline? Should there be a new underlay component that uses it?

Reply:
I defined the parameter as a reference so I could reuse it for the other endpoints (e.g. getEntity, listEntities all take an `underlayName` parameter) without redefining it inline everywhere.

Not sure what you mean about a new underlay component -- can you clarify about that please?

Reply:
I had meant an UnderlayV2 but I forgot to update this once I got there. Anyways, I understand where else you're planning to use it and I see the format of the parameter is unfortunately a bit different from what's in UnderlayV2 so it can't be reused there.

Reply:
Do you mean reuse the `underlayName` parameter definition for the `name` property definition of the `UnderlayV2` schema? I'm not sure you can do that in OpenAPI -- I thought parameters needed to be separate from the schema objects.

Reply:
I had meant that to reuse at least the description but I see now they're not quite the same format.